### PR TITLE
docs: add changelog

### PR DESCRIPTION
**Pull request by randallmorey**
Proposed changelog takes the same approach as Boundary.

### CODE REVIEW

**Comment by randallmorey** [comment]
One question @gsusmi @malnick is heading structure for UI types.  Should it be:

- version
  - UI name (admin or desktop)
    - changes/additions/fixes

OR

- version
  - changes/additions/fixes
    - UI name (admin or desktop)

**Comment by gsusmi** [comment]
I prefer version/ui name as it's much easier to identify client changes that one is interested in. How are we doing to identify features that get released to both? Something like : admin/desktop/all for UI name?

**Comment by randallmorey** [comment]
Actually, I wasn't thinking clearly.  We're not versioning for admin so it's not clear how we'd changelog it.  Let's just worry about desktop for now?

File: ui/desktop/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Boundary Desktop CHANGELOG
+
+Canonical reference for changes, improvements, and bugfixes for Boundary Desktop.
+
+
+## 0.1.0 (2021.02.16)
```
Comment (gsusmi) [inline ui/desktop/CHANGELOG.md:6]: Should this heading be `v1.0.0-alpha`?

**Review by gsusmi — APPROVED** [review]
LGTM. Ty!

**Pull request merged**